cluster/src/main/java/com/hartwig/pipeline/tools/Versions.java

```diff
@@ -21,7 +21,7 @@ public interface Versions {
     String HEALTH_CHECKER = "3.4";
     String LILAC = "1.4.2";
     String LINX = "1.23.2";
-    String ORANGE = "2.3";
+    String ORANGE = "2.4";
```

Comment:
You should use 2.4.0

Related: I don't understand where the "2.4" jar comes from on data-vm, there is no "2.4" release.